boards/arm/nucleo_f302r8/doc/nucleof302r8.rst

```diff
@@ -0,0 +1,174 @@
+.. _nucleo_f302r8_board:
+
+ST Nucleo F302R8
+################
+
+Overview
+********
+
+The STM32 Nucleo board provides an affordable and flexible way for users to try
+out new concepts and build prototypes with the STM32 microcontroller, choosing
+from the various combinations of performance, power consumption and features.
+For the compatible boards, the SMPS significantly reduces power consumption in
+Run mode.
+
+The Arduino™ Uno V3 connectivity support and the ST morpho headers allow the
+easy expansion of the functionality of the STM32 Nucleo open development
+platform with a wide choice of specialized shields.
+
+The STM32 Nucleo board does not require any separate probe as it integrates the
+ST-LINK/V2-1 debugger and programmer.
+
+The STM32 Nucleo board comes with the STM32 comprehensive software HAL library
+ together with various packaged software examples, as well as direct access to
```

Comment:
delete the leading space on this line and the next

Reply:
fixed in commit 22cf2e0d35e